docs/Tutorials/GettingStarted/Tutorial.md

```diff
@@ -118,7 +118,7 @@ Most developers wish to run F´ on embedded hardware. This is done by generating
 toolchain by providing the toolchain argument. The above invocations assume the default "native" toolchain.
 
 Here, the "raspberrypi" toolchain supplied by F´ is used. This will generate a cross-compile for the raspberry pi
-architecture.  The user should ensure to follow the Raspberry PI setup steps found at: [RPI](https://github.com/nasa/fprime/blob/master/RPI/README.md)
+architecture.  The user should ensure to follow the Raspberry PI setup steps found at: [RPI](../../../RPI/README.md)
```

Comment:
These docs are rendered as part of our documentation.  Links like this that leave the `docs` directory need to be a full url to the github path.

Reply:
Sure. didn't knew about that.